Automattic/wp-calypso
# Janitorial: Refactor nameserver components

- ES6ify
- Use Redux notices
- Use localize

Test:
- Go to nameservers view
- Use the toggle
- Set some custom NS
- Use the trash can to delete a NS from customs

Make sure no errors in console and all works.
